overlord/snapstate/snapstate.go

```diff
@@ -290,9 +290,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		confFlags |= UseConfigDefaults
 	}
 
-	configSet := ConfigureSnap(st, snapsup.Name(), confFlags)
-	configSet.WaitAll(ts)
-	ts.AddAll(configSet)
+	// we do not support configuration for bases or the "snapd" snap yet
```

Comment:
could you add TODO here about not needing maybeCore anymore

Reply:
I did https://github.com/snapcore/snapd/compare/master...mvo5:no-more-maybe-core?expand=1 which is based on this PR, so once that is landed we should be able to remove maybeCore.